vspace/src/main/java/edu/asu/diging/vspace/web/staff/DeleteSlideController.java

```diff
@@ -0,0 +1,37 @@
+package edu.asu.diging.vspace.web.staff;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import edu.asu.diging.vspace.core.exception.SlideDoesNotExistException;
+import edu.asu.diging.vspace.core.services.ISlideManager;
+
+@Controller
+public class DeleteSlideController {
+    
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+    
+    @Autowired
+    private ISlideManager slideManager;
+
+	@RequestMapping(value = "/staff/module/{id}/slide/{slideId}/{flag}", method = RequestMethod.DELETE)
+	public ResponseEntity<String> deleteSlide(@PathVariable("id") String moduleId,
+			@PathVariable("slideId") String slideId, @PathVariable("flag") String flag) {
+		try {
+			slideManager.deleteSlideById(slideId, moduleId, flag);
+		} catch (SlideDoesNotExistException slideDoesNotExistException) {
+			logger.error("Could not delete slide.", slideDoesNotExistException);
+			return new ResponseEntity<String>("Sorry, unable to delete the slide. Please try later",
```

Comment:
no, don't try again later. the slide doesn't exist, why would trying again later help with anything?